File: AD1/Assignment_1/Q8.java
/*Name:Anish More
 Name of the program:Generating nth fibonacci number
 */
import java.util.Scanner;

public class Q8 {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter the number: ");
    int num = sc.nextInt();
    int x = 0, y = 1, c;
    for (int i = 1; i < num; i++) {
      c = x + y;
      x = y;
      y = c;
    }
    System.out.println(num + "th fibonacci number is: " + x);
  }
}
/*Output 
Enter the number: 10
10th fibonacci number is: 34
 */

File: Python/Minor_Assignment_2/Q8.py
num=int(input("Enter a number:"))
while num<=0:
    num=int(input("Number should be positive.Enter number:"))
if num%2==0:
    num*=2
else:
    num**=2
print("Result:",num)


'''
Output--
Enter a number:-2
Number should be positive.Enter number:4
Result: 8
'''